be/src/util/countdown_latch.h

```diff
@@ -34,10 +34,11 @@ namespace starrocks {
 // This is a C++ implementation of the Java CountDownLatch
 // class.
 // See http://docs.oracle.com/javase/6/docs/api/java/util/concurrent/CountDownLatch.html
-class CountDownLatch {
+template <typename Lock, typename Cond>
+class GenericCountDownLatch {
```

Comment:
Why not keep the origin name CountDownLatch?

Reply:
CountDownLatch is defined as
```
using CountDownLatch = GenericCountDownLatch<std::mutex, std::condition_variable>;
```

Reply:
OK